editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -297,59 +301,42 @@ export const editorStore: StateCreator<
   },
 
   createFlow: async (teamId, newSlug, newName) => {
-    let response = (await client.mutate({
-      mutation: gql`
-        mutation CreateFlow(
-          $data: jsonb
-          $slug: String
-          $teamId: Int
-          $name: String
-        ) {
-          insert_flows_one(
-            object: { slug: $slug, team_id: $teamId, version: 1, name: $name }
-          ) {
-            id
-            data
-          }
-        }
-      `,
-      variables: {
-        name: newName,
+    const token = get().jwt;
+
+    await axios.post(
+      `${import.meta.env.VITE_APP_API_URL}/flows/create`,
+      {
+        teamId: teamId,
         slug: newSlug,
-        teamId,
+        name: newName,
+        data: { [ROOT_NODE_KEY]: { edges: [] } },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
       },
-    })) as any;
+    );
 
-    const { id } = response.data.insert_flows_one;
+    return newSlug;
+  },
 
-    response = await client.mutate({
-      mutation: gql`
-        mutation InsertOperation($flow_id: uuid, $data: jsonb) {
-          insert_operations_one(
-            object: { flow_id: $flow_id, data: $data, version: 1 }
-          ) {
-            id
-          }
-        }
-      `,
-      variables: {
-        flow_id: id,
-        data: {
-          m: { ts: 1592485241858 },
-          v: 0,
-          seq: 1,
-          src: "143711878a0ab64c35c32c6055358f5e",
-          create: {
-            data: { [ROOT_NODE_KEY]: { edges: [] } },
-            type: "http://sharejs.org/types/JSONv0",
-          },
```

Comment:
It's worth highlighting that our frontend mutation which created the operation associated with a new flow was defining more `data` than our API equivalant (which has been actively in use by `/flows/:flowId/copy` endpoint). 

We haven't ever hit issues with this before for copied flows, so I have not updated the existing API mutation. Any objections to this? 

Compare here: https://github.com/theopensystemslab/planx-new/blob/main/api.planx.uk/helpers.ts#L129